app/assets/stylesheets/components/buttons.scss

```diff
@@ -19,81 +19,59 @@
 
 // Styles
 .crayons-btn {
-  background-color: $dolphin;
-  color: $snowflake;
+  background-color: var(--button-primary-bg);
+  color: var(--button-primary-color);
 
   &--disabled,
   &[disabled] {
-    color: rgba($snowflake, 0.8);
-    background-color: $dolphin-lighter;
+    opacity: 0.6;
```

Comment:
I changed the styling for "disabled" state so we can limit number of necessary variables.